diagnostic.md

```diff
@@ -9,6 +9,8 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+primitives are data types. example of a javascript parameter is boolean.
+
```

Comment:
We are looking for a list of _all_ JavaScript primitives, and an example of _each_ one. Take a look at the lesson on JS Basics again: https://github.com/ga-wdi-boston/js-basics